src/SmartComponents/Compliance/ComplianceCard.scss

```diff
@@ -40,4 +40,8 @@
     > .pf-l-stack .pf-l-stack__item .pf-l-split .pf-l-split__item {
       font-size: var(--pf-global--FontSize--sm);
     }
-  }
\ No newline at end of file
+  }
+
+.ins-c-card__compliance {
```

Comment:
does this visually change the card?  should we apply this style to the template card (when sans actions?)

Reply:
Yeah, when there is no report button on this card or remediations, extra padding is applied. We could probably extract it somewhere. @christiemolloy and ideas on where to add it?

Reply:
umm id have to see it, want me to take this issue? @ryelo

Reply:
Sure thing. You'll notice this in both remediations and compliance if you remove the TemplateCardActions

Reply:
ok!